### DISCUSSION TITLE
loading issue

### DISCUSSION THREAD

**Discussion by dilini-liyanage** [shuding/nextra#2910] [Q&A] [answered]
When I load the home page, I include nested folders, like display as a menu, when page is loading they are displayed in opened view. I want to display them in a closed view. want to display them after clicking on the component.

![image](https://github.com/shuding/nextra/assets/121649435/d26251fe-efa8-463c-842a-ba0776157d83)

**Comment by hariria** [comment, marked as answer]
https://nextra.site/docs/docs-theme/theme-configuration#menu-collapse-level
